SpeechService/Translations.cs

```diff
@@ -50,7 +50,7 @@ public static string Power(string power)
         private static readonly Dictionary<string, string> STAR_SYSTEM_FIXES = new Dictionary<string, string>()
         {
             { "VESPER-M4", "Vesper M 4" }, // Stop Vesper being treated as a sector
-            { "Sagittarius A*", "Sagittarius A- Star" }, // Allow the * to be parsed out
+            { "Sagittarius A*", "Sagittarius A Star" }, // Allow the * to be parsed out
```

Comment:
If you test with the voice `Microsoft Hazel`, you'll find that the dash is serving an important function for that voice. It's the difference between "a" as in "a star" and "A" as in "Star A".

We could convert this to use SSML (via the `sayAsLettersOrNumbers` method in Translations.cs), but in doing so we may lose fidelity in some voices that have less built-in support for SSML (this is also possibly mitigated in Cereproc voices by commit https://github.com/EDCD/EDDI/pull/1606/commits/686b4bfb1bbbbfc355b7b21781e894ff1b455ba7).

By the way, none of the voices that I have installed currently (including one Cereproc voice) read this as "Sagittarius A Minus Star". Do you have a counter example where it does?

Reply:
Oh, that's what it was for.

Unfortunately it doesn't seem to do anything on my end — Hazel says "Sagittarius, a star" in both cases. Neither does double-quoting the "A".

> By the way, none of the voices that I have installed currently (including one Cereproc voice) read this as "Sagittarius A Minus Star". Do you have a counter example where it does?

In fact, Cereproc Heather does this.

But you're right that some sort of markup is required in place of this `-` hack, and a fallback for non-SSML-capable voices too. Which of the commonly used voices don't implement SSML (or don't do it good enough for this purpose)?

Reply:
Superseded by PR #1619.